### ISSUE TITLE
Tried using some modules from sam6d but got poor results after setting model.eval() during testing

### ISSUE THREAD

**Issue by W-QY** [JiehongLin/SAM-6D#55]
First of all thank you very much for your work! 

I tried to add the "Fine Point Matching" module in your code to my network. During the training process, the loss converged very well, but the test results were poor. After I removed "model.eval()", the model test results were very good. I would like to ask you if you know the reason?

**Comment by huassaue** [comment]
是不是batch_size设置过小
